pkg/reconciler/trigger/trigger.go

```diff
@@ -37,21 +37,17 @@ import (
 	metadataClient "github.com/google/knative-gcp/pkg/gclient/metadata"
 	"github.com/google/knative-gcp/pkg/reconciler"
 	"github.com/google/knative-gcp/pkg/reconciler/broker/resources"
+	reconcilerutilspubsub "github.com/google/knative-gcp/pkg/reconciler/utils/pubsub"
 	"github.com/google/knative-gcp/pkg/utils"
 	"knative.dev/eventing/pkg/apis/eventing/v1beta1"
 )
 
 const (
 	// Name of the corev1.Events emitted from the Trigger reconciliation process.
-	triggerReconciled         = "TriggerReconciled"
-	triggerFinalized          = "TriggerFinalized"
-	triggerReadinessChanged   = "TriggerReadinessChanged"
-	triggerReconcileFailed    = "TriggerReconcileFailed"
-	triggerUpdateStatusFailed = "TriggerUpdateStatusFailed"
-	topicCreated              = "TopicCreated"
-	subCreated                = "SubscriptionCreated"
-	topicDeleted              = "TopicDeleted"
-	subDeleted                = "SubscriptionDeleted"
+	triggerReconciled = "TriggerReconciled"
+	triggerFinalized  = "TriggerFinalized"
+	topicDeleted      = "TopicDeleted"
```

Comment:
Do we expect to switch to PubsubReconciler's delete functionality in a future PR?

Reply:
I should have just done it :)